app/packages/backend/Dockerfile

```diff
@@ -25,10 +25,13 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get update && \
     apt-get install -y --no-install-recommends libsqlite3-dev
```

Comment:
I dont think we need this RUN layer as we dont use sqlite3 in the deployed container, we use postgres. @jamescoxheadcapgemini could you confirm?

Reply:
we use postgres

Reply:
Happy to do that, but i also need to move it in the package json to a dev dependency. That requires a bit of testintg. I'll do it in a new PR after this one, so as not to mix up small code changes, with a structural change.